.github/workflows/docker_kms_sgx_tests.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: KMS SGX tests
+
+env:
+  KMS_USE_BOOTSTRAP_SERVER: true
+  KMS_USE_CERTBOT: true
+  KMS_CERTBOT_HOSTNAME: kms.sgx.ci.cosmian.dev
+  KMS_CERTBOT_EMAIL: tech@cosmian.com
+  KMS_CERTBOT_USE_TEE_KEY: abcdef0123456789    
+
+on:
+  workflow_call:
+    inputs:
+      kms-version:
+        required: true
+        type: string
+
+jobs:
+
+  run_and_test:
+
+    services:
+      kms:
+        image: ghcr.io/cosmian/enclave-kms-insecure:${{ inputs.kms-version }}
```

Comment:
Are we sure that the docker will stop if the test success or fail ? Or should we expect to stop it manually sometimes? Is there a way to prevent that if so?